scripts/dataset_processing/get_ami_data.py

```diff
@@ -20,61 +20,16 @@
 
 from nemo.collections.asr.parts.utils.manifest_utils import create_manifest
 
-# todo: once https://github.com/tango4j/diarization_annotation/pull/1 merged, we can use the same repo
-test_rttm_url = (
-    "https://raw.githubusercontent.com/tango4j/diarization_annotation/main/AMI_corpus/test/split_rttms.tar.gz"
-)
-dev_rttm_url = (
-    "https://raw.githubusercontent.com/SeanNaren/diarization_annotation/dev/AMI_corpus/dev/split_rttms.tar.gz"
-)
+rttm_url = "https://raw.githubusercontent.com/BUTSpeechFIT/AMI-diarization-setup/main/only_words/rttms/{}/{}.rttm"
+uem_url = "https://raw.githubusercontent.com/BUTSpeechFIT/AMI-diarization-setup/main/uems/{}/{}.uem"
+list_url = "https://raw.githubusercontent.com/BUTSpeechFIT/AMI-diarization-setup/main/lists/{}.meetings.txt"
```

Comment:
Can we also add text transcription for each session from speakers independently, so this script can be used by ASR folks as well.